app/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Amar Muha</title>
+  <link rel="stylesheet" href="styles/style.css">
+</head>
+
+<body>
+  <div class="wrapper">
+    <section class="header-section">
+      <header class="main-header">
+        <div class="logo">
+          Amar
+        </div>
+        <div class="toggler" id="toggler"></div>
+        <!-- <ul class="nav" id="show">
+          <li class="nav-elements">Portfolio</li>
+          <li class="nav-elements">About</li>
+          <li class="nav-elements">Contact</li>
+        </ul> -->
+      </header>
+      <div class="banner">
+        <div class="banner-header">
+          <h2>I'm Amar Glad to see you!</h2>
+        </div>
+        <div class="banner-body">
+          <p>I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hasitate to contact me.</p>
+        </div>
+        <div class="banner-footer">
+          <a href="#">LET'S CONNECT</a>
+          <div class="banner-social">
+            <div class="social"><a href="#"><img src="./assets/pngs/twitter.png"></a></div>
+            <div class="social"><a href="#"><img src="./assets/pngs/Linkedin.png" alt=""></a></div>
+            <div class="social"><a href="#"><img src="./assets/pngs/github.png" alt=""></a></div>
+          </div>
```

Comment:
- [ ] Please I suggest wrapping the icons with `ul` tag and each icons with `li` tags with each icon having a link `a` tag. This will help implement HTML semantics for users and the `a` tag to help you connect your social media icons with `href`. Please [check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Content_categories#flow_content) for more information. 😃